internal/cache/controller_obj_cache.go

```diff
@@ -1786,14 +1786,15 @@ func (c *AviObjCache) AviPopulateHttpPolicySetbyUUID(client *clients.AviClient,
 
 		}
 	}
+	tenant := getTenantFromTenantRef(*httppol.TenantRef)
```

Comment:
is there any default tenant value that needs to be set incase *httppol.TenantRef is empty? Are we sure that *httppol.TenantRef will always be non-empty?